### DISCUSSION TITLE
Upgrading undici from 5.22.0 to 6.19.0 is breaking msw post handler

### DISCUSSION THREAD

**Discussion by kjindal0802** [nodejs/undici#3333] [Q&A]
Upgrading undici from 5.22.0 to 6.19.0 is breaking msw post handler

**Comment by mcollina** [comment]
I have literally no idea. We don't support msw, you should report it to them

**Reply by kjindal0802** [reply]
@mcollina I already had an open issue there in msw  and that does not seems to breaks anything but while updating undici from 5.22.0 to 6.19.0, jest test cases breaks.
